app/models/course/virtual_classroom_settings.rb

```diff
@@ -27,6 +27,20 @@ def braincert_whiteboard_api_key=(value)
     @settings.braincert_whiteboard_api_key = value
   end
 
+  # Returns BrainCert Virtual Classroom server region
+  #
+  # @return [String] The custom or default title of virtual classrooms component
+  def braincert_server_region
+    @settings.braincert_server_region || 7 # 7 is code for Singapore
```

Comment:
Hi Weiqing,

For now it's an integer. I am not sure whether that may be changed by BrainCert side in the future though.

What would your suggestion be? Should I use integer input or just normal text input?